src/components/organisms/CardEditorDialog.tsx

```diff
@@ -15,23 +17,42 @@ interface IProps {
   text: string | undefined;
 }
 
+const barsWidgetsLabels: Record<string, any> = {
+  'accident_count_by_accident_year':
+    ['textView.fatal.plural', 'textView.severe.plural', 'textView.light.plural'],
+  'injured_count_by_accident_year':
+    ['textView.killed.plural', 'textView.severeInjured.plural', 'textView.lightInjured.plural'],
+}
+
+const barsWidgetsTitle: Record<string, string> = {
+  'accident_count_by_accident_year': 'cardEditor.showAccidents',
+  'injured_count_by_accident_year': 'cardEditor.showInjured',
+}
+
 const CardEditor: FC<IProps> = ({ isOpen, onClose, widgetName, text }) => {
   const [cardElement, setCardElement] = useState({});
   const [size, setSize] = useState(1);
+  const [barValues, setBarValues] = useState( [true, true, true] );
```

Comment:
can you use an object with field names instead of an array?

Reply:
How would you call the fields? (it is relevant to all charts so it needs to be generic)